hydra-node/src/Hydra/Ledger/Cardano/Evaluate.hs

```diff
@@ -263,6 +264,17 @@ genPointInTime = do
   let time = slotNoToUTCTime slot
   pure (slot, time)
 
+-- | Parameter here is the contestation period (cp) so we need to generate
+-- start (tMin) and end (tMax) tx validity bound such that their difference
+-- is not higher than the cp
+genSlotWithPointInTimeFromCP :: Word64 -> Gen (SlotNo, (SlotNo, UTCTime))
```

Comment:
Should: use the type `ContestationPeriod`

Also, please indicate that the returned slots are validity bounds. So maybe calling this `genValidityBoundsFromContestationPeriod`?